### PR TITLE
Rename lang to partiql-lang. Move partiql-extensions into cli

### PR DESCRIPTION
**Pull request by RCHowell**
## Description

Cleaning up structure. App subdirectory and partiql-extensions were not necessary. Rename lang to partiql-lang for consistency with other subprojects. Cleaned up github workflows as well.

## Other Information
- Updated Unreleased Section in CHANGELOG: **[YES/NO]**
No

- Any backward-incompatible changes? **[YES/NO]**
No, just a dir structure change.

- Any new external dependencies? **[YES/NO]**
No

## License Information

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .github/workflows/docs-to-gh-wiki.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@ jobs:
     - name: Push Wiki Changes
       uses: Andrew-Chen-Wang/github-wiki-action@v3
       env:
-        WIKI_DIR: docs/
+        WIKI_DIR: docs/wiki
```
Comment (johnedquinn) [inline .github/workflows/docs-to-gh-wiki.yml:18]: This needs to end in a `/` as it uses rsync.

File: partiql-cli/src/main/kotlin/org/partiql/cli/functions/QueryDDB.kt [hunk 2]
```diff
@@ -1,17 +1,4 @@
...
- * A copy of the License is located at:
- *
- *      http://aws.amazon.com/apache2.0/
- *
- * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
- * language governing permissions and limitations under the License.
- */
-package org.partiql.extensions.cli.functions
+package org.partiql.cli.functions
```
Comment (johnedquinn) [inline partiql-cli/src/main/kotlin/org/partiql/cli/functions/QueryDDB.kt:1]: Missing the copyright info.

File: partiql-cli/src/test/kotlin/org/partiql/cli/functions/QueryDDBTest.kt [hunk 3]
```diff
@@ -1,23 +1,10 @@
...
- * A copy of the License is located at:
- *
- *      http://aws.amazon.com/apache2.0/
- *
- * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
- * language governing permissions and limitations under the License.
- */
-package org.partiql.extensions.cli.functions
+package org.partiql.cli.functions
```
Comment (johnedquinn) [inline partiql-cli/src/test/kotlin/org/partiql/cli/functions/QueryDDBTest.kt:1]: Copyright info.

**Review by johnedquinn — CHANGES_REQUESTED** [review]
The only blocking comment is using `/` at end of `WIKI_DIR`. Beyond that, I do have one non-blocking question about moving `upgrades`: why place it in `docs` versus `examples`?

File: .github/workflows/docs-to-gh-wiki.yml (hunk 1 above)
Comment (RCHowell) [reply .github/workflows/docs-to-gh-wiki.yml:18]: nice catch

**Review by johnedquinn — APPROVED** [review]

**Pull request merged**